### PR TITLE
CLOUDP-57866: Move PCGC into github.com/mongodb & rename it

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ [CLOUDP-57866](https://jira.mongodb.org/browse/CLOUDP-57866)

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/mocks/mock_owners.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ package mocks
 
 import (
 	gomock "github.com/golang/mock/gomock"
-	cloudmanager "github.com/mongodb-labs/pcgc/cloudmanager"
+	opsmngr "github.com/mongodb/go-client-mongodb-ops-manager/opsmngr"
```
Comment (robcarlan-mlab) [inline internal/mocks/mock_owners.go]: (and here!)

Comment (robcarlan-mlab) [inline internal/mocks/mock_owners.go]: (and here!)

File: internal/mocks/mock_automation.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ package mocks
 
 import (
 	gomock "github.com/golang/mock/gomock"
-	cloudmanager "github.com/mongodb-labs/pcgc/cloudmanager"
+	opsmngr "github.com/mongodb/go-client-mongodb-ops-manager/opsmngr"
```
[the remark this answers is not in the document]
Comment (gssbzn) [reply internal/mocks/mock_automation.go]: mocks are auto generated, not to be edited same the other mock

[the remark this answers is not in the document]
Comment (robcarlan-mlab) [reply internal/mocks/mock_automation.go]: 👍

[the remark this answers is not in the document]
Comment (robcarlan-mlab) [reply internal/mocks/mock_automation.go]: 👍

**Pull request merged**